docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -0,0 +1,259 @@
+
+[discrete]
+[[view-the-number-of-transactions-per-day]]
+=== View the number of transactions per hour
+
+To determine the number of orders made per hour, create a bar chart, then add it to the dashboard.
+
+. Set the <<set-time-filter,time filter>> to *Last 30 days*.
+
+. From the *Available fields* list, drag and drop *Records* to the visualization builder.
++
+[role="screenshot"]
+image::images/lens_advanced_1_1.png[Added records to the workspace]
+
+. Change the *Vertical axis* title and display the number of orders per day. 
+
+.. From the editor, click *Count of Records*.
+
+.. In the *Display name* field, enter `Number of orders`.
+
+.. Click *Add advanced options > Normalize by unit*.
+
+.. From the *Normalize by unit* dropdown, select *per hour*, then click *Close*.
+
+. To hide the *Horizontal axis* label, open the *Bottom Axis* menu, then deselect *Show*.
++
+[role="screenshot"]
+image::images/lens_advanced_1_1_2.png[Bottom axis menu]
++
+You have a bar chart that shows you how many orders were made at your store per hour.
++
+[role="screenshot"]
+image::images/lens_advanced_1_2.png[Orders per day]
+
+. Click *Save and return*.
+
+[discrete]
+[[view-the-cumulative-number-of-products-sold-over-time]]
+=== View the cumulative number of products sold on weekends
+
+To determine the number of orders made only on Saturday and Sunday, create an area chart, then add it to the dashboard.
+
+. Open *Lens*.
+
+. From the *Chart Type* dropdown, select *Area*.
++
+[role="screenshot"]
+image::images/lens_advanced_2_1_1.png[Chart type menu with Area selected]
+
+. Configure the cumulative sum of the store orders.
+
+.. From the *Available fields* list, drag and drop *Records* to the visualization builder.
+
+.. From the editor, click *Count of Records*.
+
+.. From *Select a function*, click *Cumulative sum*.
+
+.. In the *Display name* field, enter `Cumulative orders during weekend days`, then click *Close*.
+
+. Filter the results to display the data for only Saturday and Sunday.
+
+.. From the editor, click the *Drop a field or click to add* field for *Break down by*. 
+
+.. From *Select a function*, click *Filters*.
+
+.. Click *All records*.
+
+.. In the *KQL* field, enter `day_of_week : "Saturday" or day_of_week : "Sunday"`, then press Return.
++
+[role="screenshot"]
+image::images/lens_advanced_2_1.png[Filter aggregation to filter weekend days]
+
+. To hide the legend, open the *Legend* menu, then click *Hide*.
++
+[role="screenshot"]
+image::images/lens_advanced_2_2_1.png[Legend menu]
++
+You have an area chart that shows you how many orders your store received during the weekend.
++
+[role="screenshot"]
+image::images/lens_advanced_2_2.png[Line chart with cumulative sum of orders made on the weekend]
+
+. Click *Save and return*.
+
+[discrete]
+[[add-a-data-layer]]
+=== View the key percentiles of product prices over time
+
+To view the price distribution of products sold over time, create a percentile chart, then add it to the dashboard.
```

Comment:
I'm leaving some very similar feedback here because I have a template in mind for how we can make the Lens docs as useful as possible.

```
=== Section heading that is focused on a user task, that I would send as an answer in Discuss or Slack

The first sentence should explain the _context_ for this task and any background knowledge that the user needs, without restating the heading. The second sentence should describe any choices the user needs to make during this process, since if there are no choices to be made then the feature is too simple to have a tutorial.

This sentence describes the setup for both choices:

1. Do something

If your data looks like the first choice, do this:

1. 

But if it looks like the second choice, do this:

1.
```

As written, I find these tutorials to be _very hard to read_ because the steps are listed without context.

cc @KOTungseth

Reply:
I am not sure I agree with the statement 'since if there are no choices to be made then the feature is too simple to have a tutorial'. 

For example, there's only one way of focusing drag and drop, and yet we're explaining it because it's not trivial. 

Also, during the kibana data analyst course, we learn things like moving average (despite having examples of other pipeline metrics before) because sometimes that's what user wants but cannot call it so we explain the feature in the context of what we want to build. This tutorial is story-based and it omits some paths you might choose to get the same or similar result, so maybe it's something we could rethink (adding the context you mention), but I wouldn't delete parts of it just because there is only one way of doing it.